hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/message/HL7Message.java

```diff
@@ -0,0 +1,135 @@
+package ca.bc.gov.hlth.hnsecure.message;
+
+public class HL7Message {
+
+	public String segmentIdentifier = "MSH";
```

Comment:
Should this be private and just use getters/setters like the rest of the variables